app/register/index.js

```diff
@@ -0,0 +1,67 @@
+const submitButton = document.getElementById('submit-button');
+const passwordElement = document.getElementById('password');
+
+const clearWarningMessages = () => {
+	const warningMessages = document
+		.getElementById('warning-message-container')
+		.getElementsByTagName('P');
+	const numberOfMessages = warningMessages.length;
+
+	for (let index = 0; index < numberOfMessages; index++) {
+		warningMessages[index].classList.add('hidden');
+	}
+};
+
+const clearInputValue = (event) => {
+	event.target.value = '';
+};
+
+const enableSubmitButton = () => {
+	submitButton.disabled = false;
+};
+
+const disableSubmitButton = () => {
+	submitButton.disabled = true;
+};
+
```

Comment:
Not sure about putting these in their own functions. I quite liked that the functions that use this are really easy to read but equally we could just add `submitButton.disabled = true;` in place of calling the function.
What do you think, Kenny?

Reply:
I think this looks nice! I think either way is fine, for the reasons you've described here. Pros and cons to each